crossbeam-epoch/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Version 0.7.3
+
+- Bump the minimum required version to 1.28.
```

Comment:
Isn't it considered a breaking change?  (I'm asking this because I don't know the convention of Rust libraries...)

Reply:
I would say it is a breaking change. In #412 we had this exact issue, where `lazy_static` dropped support for older rustc in a minor version bump, which caused a little trouble.

Reply:
There are many crates that do this in patch version (lazy_static, [pkg-config](https://github.com/rust-lang/pkg-config-rs/blob/master/CHANGELOG.md#0315---2019-07-25), [serde_derive](https://github.com/serde-rs/serde/releases/tag/v1.0.99), [failure](https://github.com/rust-lang-nursery/failure/pull/319#issuecomment-531891937), etc.), but I think it makes sense to consider it a breaking change.

Reply:
Since it's a breaking change, I would like to bump the minor version...

Having said that, I also hesitantly agree not to bump the version following the other projects' precedence.  I' marking this issue resolved...